python/lsst/obs/decam/decamMapper.py

```diff
@@ -29,21 +29,21 @@
 
 np.seterr(divide="ignore")
 
-class DecamInstcalMapper(CameraMapper):
+class DecamMapper(CameraMapper):
     packageName = 'obs_decam'
 
     def __init__(self, inputPolicy=None, **kwargs):
         policyFile = pexPolicy.DefaultPolicyFile(self.packageName, "DecamMapper.paf", "policy")
         policy = pexPolicy.Policy(policyFile)
 
-        super(DecamInstcalMapper, self).__init__(policy, policyFile.getRepositoryPath(), **kwargs)
+        super(DecamMapper, self).__init__(policy, policyFile.getRepositoryPath(), **kwargs)
 
-        afwImageUtils.defineFilter('u', lambdaEff=350)
-        afwImageUtils.defineFilter('g', lambdaEff=450)
-        afwImageUtils.defineFilter('r', lambdaEff=600)
-        afwImageUtils.defineFilter('i', lambdaEff=750)
-        afwImageUtils.defineFilter('z', lambdaEff=900)
-        afwImageUtils.defineFilter('y', lambdaEff=1000, alias='Y')
+        afwImageUtils.defineFilter('u', lambdaEff=350, alias=['u DECam c0006 3500.0 1000.0'])
+        afwImageUtils.defineFilter('g', lambdaEff=450, alias=['g DECam SDSS c0001 4720.0 1520.0'])
+        afwImageUtils.defineFilter('r', lambdaEff=600, alias=['r DECam SDSS c0002 6415.0 1480.0'])
+        afwImageUtils.defineFilter('i', lambdaEff=750, alias=['i DECam SDSS c0003 7835.0 1470.0'])
+        afwImageUtils.defineFilter('z', lambdaEff=900, alias=['z DECam SDSS c0004 9260.0 1520.0'])
+        afwImageUtils.defineFilter('y', lambdaEff=1000, alias=['Y','Y DECam c0005 10095.0 1130.0'])
```

Comment:
No, no, no!  That's the wrong way!

The DECam "y" filter is NOT an NIR filter --- yeah, we're talking about 1um, but it's on silicon, so the shape of the bandpass is very different from a "Y" filter on HgCdTe.  DECam documentation may call it "Y" (and many people do, with other instruments too), but it's a dangerous confusion to encourage.  The convention has been lower case wide-band interference filters over silicon, upper-case for other detectors operating into the NIR.  Please don't add to the confusion!